### ISSUE TITLE
[Opera Presto] FLV 240p no longer available

### ISSUE THREAD

**Issue by StarWolf3000** [gantt/downloadyoutube#78]
I know, no one ever probably is using this resolution, but it just bugs me, that, at least on Opera Presto, FLV 240p is no longer available, only MP4 360p and MP4 720p are listed. A week ago or so it was available as usual.

Has 240p moved to DASH? Because it is still available in the quality selection.

**Comment by was8309** [comment]
same for firefox 47 on windows 7. ( and unrelated here but vlc only seems to convert flv to mp3 successfully) . thanks

**Comment by gantt** [comment]
Yes, it looks like YouTube removed the links for FLV files (fmt=5). It's not a browser-specific issue. If you have a recent browser, the HTML5 player doesn't use the legacy download URLs, they're only for the old Flash player.
